apps/protocol-frontend/src/contexts/UserContext.tsx

```diff
@@ -65,16 +70,17 @@ export const UserContextProvider: React.FC<UserContextProps> = ({
     [],
   );
   const [allDaos, setAllDaos] = useState<UIGuild[]>([]);
+  const [userContributionsDateRangeCount, setUserContributionsDateRangeCount] =
+    useState<any>(null);
```

Comment:
The any should also be removed here. We can add a ui type and then this will be the ui type and null